ClinGen/clincoded
# Connects to #1426. Connects to #1427. Connects to #1428. Updates to "Predictors" tab in VCI

**Technical Notes:**
Example ClinVar ID for testing: **55487**

Common steps to test #1426, #1427 and #1428:
1\. Select a variant in the VCI.
2\. Click on the “Predictors” tab.

Additional steps to test #1426:
3. Confirm that the predictor names (first column) are functional links going to their assigned URLs.

Additional steps to test #1427:
3. Click on the info icon next to the “Prediction” column header to bring up a popover window.
4. Confirm that the table matches the data provided in the ticket.

Additional steps to test #1428:
3. Confirm that the “Pathogenicity Threshold” column is present and the data matches the ticket.
